tests/test_notifications.py

```diff
@@ -681,7 +685,7 @@ def test_serialize_node_level_event_that_adopts_parent_settings(self):
         self.project_subscription.save()
         self.node.add_contributor(contributor=user, permissions=['read'])
         self.node.save()
-        node_subscriptions = [x for x in utils.get_all_node_subscriptions(user, self.node)]
+        node_subscriptions = [x for x in utils.get_all_node_subscriptions(user, self.node) if x.event_name != 'mailing_list_events']
```

Comment:
`node_subscriptions` is an empty list here. Seems like we can remove this line entirely.

```
(Pdb) l
685             self.project_subscription.save()
686             self.node.add_contributor(contributor=user, permissions=['read'])
687             self.node.save()
688             node_subscriptions = [x for x in utils.get_all_node_subscriptions(user, self.node) if x.event_name != 'mailing_list_events']
689             import pdb; pdb.set_trace()
690  ->         data = utils.serialize_event(user=user, event_description='comments',
691                                          subscription=node_subscriptions, node=self.node)
692             expected = {
693                 'event': {
694                     'title': 'comments',
695                     'description': constants.NODE_SUBSCRIPTIONS_AVAILABLE['comments'],
(Pdb) node_subscriptions
[]
(Pdb)
```